core/src/main/scala/cats/syntax/partialOrder.scala

```diff
@@ -8,7 +8,7 @@ trait PartialOrderSyntax extends EqSyntax {
     new PartialOrderOps[A](a)
 }
 
-final class PartialOrderOps[A](lhs: A)(implicit A: PartialOrder[A]) {
+final class PartialOrderOps[A](lhs: A) {
```

Comment:
Can someone who knows machinist better than I do weigh in on whether this change is fine (I suspect so, but I want to make sure). cc @non @johnynek

Reply:
I have no idea about the macro expansion in machinist.

I am not sure that is what is going on here anyway. I think it may be to prevent this class from being allocated without a PartialOrder.

We could I'm not sure if this is even tested sadly.

Reply:
This seems no longer relevant, since macros have been now removed in preparation for Dotty.